clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -100,12 +100,12 @@ public Options options() {
     options.addOption(createArgOption(AUTO, "column value auto-increments (true/false)"));
     options.addOption(createArgOption(DEFAULT, "default column value"));
     options.addOption(createSimpleOption("o", OWNER, "display entity owner"));
-    options.addOption(createArgOption("r", ROLE, "role name"));
     options.addOption(createArgOption(COLUMNFILE, "CSV file describing columns"));
 
     // Properties and tags can have multiple values
```

Comment:
This comment still needs to be changed

Reply:
Thank you Justin, I update the comment to `Options that support multiple values` and remove redundant comments

> As I said above, saying something like "Options that support multiple values" might be better. Once you have more than two things, making it more generic is better. If you were to change it your way, something like this would be better: "properties, tags and roles...". either would be fine.